ethers-solc/src/compile/project.rs

```diff
@@ -343,7 +344,10 @@ fn compile_sequential(
             input.sources.len(),
             input.sources.keys()
         );
-        let output = solc.compile(&input)?;
+
+        report::solc_spawn(&solc, &version, &input);
+        let output = solc.compile_exact(&input)?;
```

Comment:
why compile -> compile_exact?

Reply:
drive by fix to make it the same as in the parallel version, since the input already includes the sources and all its imports we want to limit the output to exactly those.